test/persistence-hooks.suite.js

```diff
@@ -2453,32 +2431,15 @@ module.exports = function(dataSource, should, connectorCapabilities) {
               if (dataSource.connector.replaceOrCreate) {
                 observedContexts.should.eql(aTestModelCtx(expected));
               } else {
-                // TODO: Please see loopback-datasource-juggler/issues#836
-                //
-                // loaded hook is triggered twice in non-atomic version:
-                // 1) It gets triggered once by "find()" in this chain:
-                //    "replaceORCreate()->findOne()->find()",
-                //    which is a bug; Please see this ticket:
-                //    loopback-datasource-juggler/issues#836.
-                // 2) It, also, gets triggered in "replaceAttributes()"
-                //    in this chain replaceORCreate()->replaceAttributes()
-                observedContexts.should.eql([
-                  aTestModelCtx({
-                    data: {
-                      id: existingInstance.id,
-                      name: 'first',
-                    },
-                    isNewInstance: false,
-                    options: { notify: false },
-                  }),
```

Comment:
Can we get rid of `if (dataSource.connector.replaceOrCreate)` and use the same code as in the other branch?